app/scripts/modules/kubernetes/src/v2/manifest/showManifestDetails.component.ts

```diff
@@ -0,0 +1,81 @@
+import { module, IComponentOptions, IController } from 'angular';
+import { trim } from 'lodash';
+import { ReactInjector } from 'core/reactShims';
+
+function stripQuotes(str: string): string {
```

Comment:
These functions seem slightly odd to me in the sense that you're not exporting them and they're not private methods on the controller class.